stdlib/SparseArrays/src/sparsematrix.jl

```diff
@@ -136,7 +136,7 @@ julia> nonzeros(A)
  2
 ```
 """
-nonzeros(S::SparseMatrixCSC) = S.nzval
+nonzeros(S::SparseMatrixCSC) = getfield(S, :nzval)
```

Comment:
I'm using `getfield` so that I can "turn off" `getproperty` during the test:

https://github.com/JuliaLang/julia/blob/b4b9d3e4e5fa29bc20492d7b88e1240920db6677/stdlib/SparseArrays/test/forbidproperties.jl#L1-L3